apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -135,6 +135,42 @@ describe('UserService', () => {
 		});
 	});
 
+	describe('findByIdOrNull', () => {
+		describe('when a user with this id exists', () => {
+			const setup = () => {
+				const user: UserDO = userDoFactory.buildWithId();
+
+				userDORepo.findByIdOrNull.mockResolvedValue(user);
+
+				return {
+					user,
+				};
+			};
+
+			it('should return the user', async () => {
+				const { user } = setup();
+
+				const result: UserDO | null = await service.findByIdOrNull('id');
+
+				expect(result).toEqual(user);
+			});
+		});
+
+		describe('when a user with this id does not exist', () => {
+			const setup = () => {
+				userDORepo.findByIdOrNull.mockResolvedValue(null);
+			};
+
+			it('should return null', async () => {
+				setup();
+
+				const result: UserDO | null = await service.findByIdOrNull('id');
```

Comment:
please use new ObjectId().toHexString() or an angry cedric will comment on this later